system/lib/update_libcxx.py

```diff
@@ -60,6 +70,32 @@ def main():
   shutil.copy2(os.path.join(libcxx_dir, 'CREDITS.TXT'), local_root)
   shutil.copy2(os.path.join(libcxx_dir, 'LICENSE.TXT'), local_root)
 
+  # We don't use frozen c++03 headers for now
+  shutil.rmtree(os.path.join(local_inc, '__cxx03'))
+
+  # libcxx includes headers from LLVM's libc
+  libc_upstream_dir = os.path.join(llvm_dir, 'libc')
+  assert os.path.exists(libc_upstream_dir)
+  libc_local_dir = os.path.join(script_dir, 'llvm-libc')
+
+  copy_dirs = [
+      ('hdr',),
+      ('include', 'llvm-libc-macros'),
+      ('include', 'llvm-libc-types'),
+      ('shared',),
+      ('src', '__support'),
+  ]
```

Comment:
Should these go at the top level with the the file lists like preserve_files and excludes?

Reply:
Done (Changed the variable name to `libc_copy_dirs`)